backend/package.json

```diff
@@ -7,6 +7,7 @@
     "start": "npm run lint & node ./server.js",
     "dev": "npm run lint & nodemon ./server.js",
     "lint": "eslint . --ext .js"
+    
```

Comment:
please remove the extra space here.

Reply:
@furkanmutlu 
 Its good you went thru my code, I appreciate the feedback. Will do these changes in the next few days.

Reply:
@furkanmutlu 
I made the changes, have another look

Reply:
did you push your changes?

Reply:
I pushed it and it does show up on my branch, but it didn't update with the pull request for some reason.
I will try to figure it out today.

Reply:
Yes, iIdid push, you can see the changes in the commits tab called 'refactor'

Reply:
https://github.com/furkanmutlu/fullstack-image-app/pull/7/commits/a820500afddc372e82257ec123d0a4c9b9acd858

Reply:
@furkanmutlu 
did you get my last message?